auth-server/src/services/auth.ts

```diff
@@ -198,16 +199,17 @@ export default class AuthService {
 		const decoded = jwt.verify(token, publicKey)
 
 		if (isNaN(decoded.sub)) {
-			throw new Error('Invalid user id')
+			throw new ForbiddenError('Invalid user id')
```

Comment:
Shouldn't this rather be a `AuthenticationError` because it comes from the token?